### DISCUSSION TITLE
As fichas para a Exportação

### DISCUSSION THREAD

**Discussion by Angellorye** [YuriAlessandro/gerador-ficha-tormenta20#371] [Geral]
Para o Foundry VTT, além de estar desatualizada, não estão mais acompanhando o JdA pelo modelo base de distibuição de pontos. Eu gosto MUITO desse site e gostaria de continuar usando ele em minhas campanhas futuras, ele quebra um galho bem grande na hora de fazer NPC's por exemplo. 

Então a pergunta é, ha algum plano de atualizar a ficha do site pra o novo modelo de foundry vtt? o .json NÃO funciona :3
Ficha de Exportação do site
![image](https://github.com/user-attachments/assets/83dc7ee0-28c7-4ef7-83ca-cb4daf02aff4)
Importação
![image](https://github.com/user-attachments/assets/21b7e237-964a-42b5-b130-e391cc8a6d5b)
Erro
![image](https://github.com/user-attachments/assets/929fae15-258c-4e2e-9d07-abb67aacdfb3)

Eu acho que desde a atualização pro jogo do ano que isso não funciona mais...

**Comment by YuriAlessandro** [comment]
Corrigi isso.

De bônus, agora também tá incluindo a classe e o nível. O cálculo de PV e PM provavelmente vai estar 99% certo (a não ser que algum poder altere isso).

Ainda não importa itens, poderes e magias, mas já é um grande adianto (e fazer isso é difícil pq a gente precisa converter na exportação tudo para ficar no formato que o Foundry espera, não é muito simples).

De qualquer forma, acho que continua sendo um ótimo quebra-galho.
